client/src/components/locations/LocationDetail/LocationDetail.css

```diff
@@ -0,0 +1,20 @@
+.loc-detail-container {
+  display: flex;
+  flex-direction: column;
+  padding: 20px;
+}
+
+.loc-title {
```

Comment:
@HaseemUlHaq please remove custom css and try to reusable the css provided with bootstrap and antd rather than writing each style yourself